scripts/ldap/base_configure_ldap.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+import sys
+import os
+import ldap
+import glob
+from shutil import copyfile
+from ldap.ldapobject import SimpleLDAPObject
+
+usage = "USAGE: python3 base_configure_ldap.py [ldapadminpassword] [rootdnpassword] [usedefaultpassword] [skipssl] [forceclean]"
+totalArguments = len(sys.argv)
+
+defaultpasswd=False
+usessl=True
+forceclean=False
+LDAPADMINPASS=None
+ROOTDNPASSWORD=None
+defaultpasswds="ldapadmin"
+
+if totalArguments <= 1 :
+    print('Too few arguments.' , usage)
+    sys.exit()
+if totalArguments > 1 :
+    LDAPADMINPASS =  sys.argv[1]
+if totalArguments > 2 :
+    ROOTDNPASSWORD = sys.argv[2]
+if totalArguments > 3 and sys.argv[3] == usedefaultpassword :
```

Comment:
is it the case here usedefaultpassword should be given as third argument only?

Reply:
Yes.Order needs to be preserved

Reply:
can we use argument parser here something like -p password like this.....order need not be preserved and one more query sys.arg[3] we are checking for userpassword it has to be always at 3rd position?

Reply:
YEs.Order needs to be preserved

Reply:
I need to check on argument parser as I am not much aware of python.

Reply:
better if we use argument parser,this will be hard coded....otherwise and userpassword has to be always 3rd argument what if i don;t pass first two...and pass userdefaultpassword as first it won;t process.we have used argument parser in s3iamcli i guess u can take reference

Reply:
@abhilekh268 Implemented argument parser. Working fine in any order now. Check 1s.